pkg/geo/geomfn/topology_operations_test.go

```diff
@@ -490,3 +490,44 @@ func TestUnaryUnion(t *testing.T) {
 		})
 	}
 }
+
+func TestMinimumRotatedRectangle(t *testing.T) {
+	tests := []struct {
+		name string
+		arg  geo.Geometry
+		want geo.Geometry
+	}{
+		{
+			"empty multipoint",
+			geo.MustParseGeometry("MULTIPOINT EMPTY"),
+			geo.MustParseGeometry("POLYGON EMPTY"),
+		},
+		{
+			"multipoint",
```

Comment:
can you be more specific about the `MULTIPOINTs` these are testing?
Also, would be nice to test other shape types too (point, linestring, polygon, multipolygon, multilinestring, geometrycollection)